### ISSUE TITLE
Cannot parse most sites in php

### ISSUE THREAD

**Issue by nerdunit** [James-LG/Skyscraper#9]
The html parser will print
"Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /
Unknown HTML symbol /"
On most php based website, example code

```
    let url = "https://www.ladybirdeducation.co.uk/";
    let client = reqwest::blocking::Client::new();
    let res = client
        .get(url).send()?;
    let body = res.text()?;
    println!("body:{}",(&body).as_str());
    let document = html::parse(body.as_str())?;
    println!("parsed");
    Ok(())
```

**Comment by James-LG** [comment]
PHP is rendered into HTML server-side so your error must be from something else. Regardless I will investigate based on the URL you gave.

**Comment by James-LG** [comment]
I believe your issue was actually related to a `<` symbol in a <script> tag. Fix is released in v0.3.1, reopen the issue if your problems persist.
